src/jit/bitsetasshortlong.h

```diff
@@ -664,9 +664,11 @@ void BitSetOps</*BitSetType*/ BitSetShortLongRep,
                /*BitSetTraits*/ BitSetTraits>::ClearDLong(Env env, BitSetShortLongRep& bs)
 {
     assert(!IsShort(env));
-    // Recall that ClearD does *not* require "bs" to be of the current epoch.
-    // Therefore, we must allocate a new representation.
-    bs = MakeEmptyArrayBits(env);
```

Comment:
Agreed. We could replace it with a new Clear() function that is non-destructive.

Now, it's possible that all the ClearD functions today can use the new semantics, meaning we don't need a new Clear function. It's just very suspicious that is true, given the huge comments about why ClearD is (currently) special.